src/api/checkout.js

```diff
@@ -24,4 +24,28 @@ app.post('/', (req, res) => {
     });
 });
 
+app.post('/longterm', (req, res) => {
+    if (typeof req.body.studentId !== 'number') {
+        return res.failureJson('A student id is required.');
+    }
+    if (typeof req.body.equipmentAddresses !== 'object'
+        || typeof req.body.equipmentAddresses[0] !== 'string') {
```

Comment:
Why only check the first? This would allow `['potato', 42]` but reject `[42, 'potato']`.